### ISSUE TITLE
What is the difference between markdown-it plugin and Builder API？

### ISSUE THREAD

**Issue by social-uni** [antfu/vite-plugin-md#116]
### Clear and concise description of the problem

May I ask what is the difference between markdown-it plugin and Builder API, I want to do some editing work on html while generating html, which one should I use

### Suggested solution

May I ask what is the difference between markdown-it plugin and Builder API, I want to do some editing work on html while generating html, which one should I use

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by yankeeinlondon** [comment]
have you looked at docs folder in the repo?

**Comment by yankeeinlondon** [comment]
actually i'm not sure I understand the question about "editing on html" ... I'm going to close for now but if you can give me a better idea on what you mean then feel free to re-open.

**Comment by social-uni** [comment]
> have you looked at docs folder in the repo?

No, I haven't read it, I just read the readme in the root directory, I'll go check it out, thank you
